nuxeo-jsf/nuxeo-platform-forms-layout-io/src/test/java/org/nuxeo/ecm/platform/layout/service/TestLayoutExport.java

```diff
@@ -109,12 +109,10 @@ public void testLayoutTypeImport() throws Exception {
     }
 
     protected void checkLayoutTypeImport(String filename, boolean isCompat) throws Exception {
-        JSONObject json = null;
+        JSONObject json;
         try (InputStream in = new FileInputStream(FileUtils.getResourcePathFromContext(filename))) {
             byte[] bytes = IOUtils.toByteArray(in);
-            if (bytes.length != 0) {
-                json = JSONObject.fromObject(new String(bytes, "UTF-8"));
-            }
+            json = JSONObject.fromObject(new String(bytes, "UTF-8"));
```

Comment:
StandardCharsets.UTF_8 ?

Reply:
Yes but for another (global) refactoring. I'm focusing on issues reported currently by SpotBugs.